package.json

```diff
@@ -4,9 +4,11 @@
   "description": "A handsome dashboard framework written in javascript",
   "main": "app.js",
   "scripts": {
-    "start": "node app.js",
-    "postinstall": "webpack -p --progress --colors",
+    "start": "babel-node app.js",
+    "postinstall": "npm run build",
```

Comment:
I don't believe there is much of a difference in this case, but I don't know the best way to go about it. I guess my thought was that npm is always there, whereas someone might be choosing to not use yarn (likewise for ci, coveralls, etc).

I also don't *think* there is any difference between yarn vs npm in running tasks- But I could change it if you prefer.